test/queries/tpc-ds-01.sql

```diff
@@ -0,0 +1,22 @@
+with customer_total_return as
```

Comment:
see the corresponding Hyrise PR. Any chance we can format this?

Reply:
Ok, have not processed any of those files. Just copied them over from @mweisgut. Do you know of a CLI tool for this job? These web formatters don't allow for batch processing.

Reply:
I found [sqlformat](http://manpages.ubuntu.com/manpages/trusty/man1/sqlformat-3.1.html) but I haven't formatted the queries yet.

Reply:
https://github.com/hyrise/hyrise/pull/1679/commits/a796a936af09d9ba40fa377219cd81ea3e325aba

Reply:
Nice, thanks a lot @mweisgut 👍